app/models/concerns/vacancy_job_summary_validations.rb

```diff
@@ -3,6 +3,18 @@ module VacancyJobSummaryValidations
 
   included do
     validates :job_summary, presence: true
-    validates :about_school, presence: true
+    validate :about_school_must_not_be_blank
+  end
+
+  def about_school_must_not_be_blank
+    if defined?(vacancy) && vacancy.job_location == 'central_office'
```

Comment:
For reviewers: `if defined?(vacancy)` is here because `vacancy` is initialized by `VacancyForm#initialize`, and creating a vacancy (e.g. with FactoryBot) runs the validations separately.

Reply:
I prefer:
```
if vacancy&.job_location == 'central_office'
```

Reply:
Unfortunately, this throws an error because vacancy is not defined. I might try to pass in an argument to this method. What do you think about that tactic?

Reply:
I ended up using `begin` `rescue NameError`